lib/ddtrace/buffer.rb

```diff
@@ -4,8 +4,8 @@
 # Trace buffer that accumulates traces for a consumer.
 # Consumption can happen from a different thread.
 module Datadog
-  # Buffer that stores objects. The buffer has a maximum size and when
-  # the buffer is full, a random object is discarded.
+  # Bounded buffer used to store profiling events.
```

Comment:
This comment doesn't seem correct, as the buffer is used by the tracer as well, not only profiling.

Reply:
Good catch. I totally missed that this is something shared with other bits of ddtrace. Reverted in https://github.com/DataDog/dd-trace-rb/pull/1443 .